scripts/bootstrap.sh

```
@@ -0,0 +1,267 @@
+        [[ -d "$repo_backup_dir" ]] || continue
+        repo_name="$(basename "$repo_backup_dir")"
+        repo_path="$REPOS_DIR/$repo_name"
+
+        if [[ ! -d "$repo_path/.git" ]]; then
+            skipped "private files: $repo_name — repo not cloned yet"
+            SKIPPED_REPOS+=("$repo_name")
+            continue
+        fi
+
+        while IFS= read -r src_file; do
+            rel_path="${src_file#$repo_backup_dir}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)